pkg/moq/moq.go

```diff
@@ -291,3 +318,27 @@ var templateFuncs = template.FuncMap{
 		return strings.ToUpper(s[0:1]) + s[1:]
 	},
 }
+
+func gopaths() []string {
+	return strings.Split(os.Getenv("GOPATH"), string(filepath.ListSeparator))
+}
+
+// StripGopath teks the directory to a package and remove the gopath to get the
+// canonical package name.
+func StripGopath(p string) string {
+	for _, gopath := range gopaths() {
+		p = strings.TrimPrefix(p, path.Join(gopath, "src")+"/")
+	}
+	return p
+}
```

Comment:
In order to keep the change small, I would move

- `func gopaths`
- `func StripGopath`

back into `pkg/moq/importer.go` and also restore the copyright notice.

Reply:
I've re-added the copyright but I think it's a bit weird to have the importer file there since it won't actually contain any code related to importing anything.